.zenodo.json

```diff
@@ -94,6 +94,11 @@
       "name": "Appelhoff, Stefan",
       "orcid": "0000-0001-8002-0877"
     },
+    {
+      "affiliation": "Western University, London, ON, CA",
+      "name": "Van Dyken, Peter",
+      "orcid": "0000-0001-7357-5815"
```

Comment:
@pvandyken Please review this entry and make any suggestions.

Reply:
That's correct, thanks!